corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -81,6 +83,43 @@ def start_reindex(self, cname, reindex_batch_size=1000, requests_per_second=None
             + f"\"grep '{task_number}.*ReindexResponse' /opt/data/elasticsearch*/logs/*.log\""
             + "\n\n")
 
+    def reindex_partial_index(self, cname, domain):
+        adapter = doc_adapter_from_cname(cname)
+        if not adapter.parent_index_name:
+            raise IndexNotPartialException(f"Adapter for {cname} is not a partial index")
+
+        source_index = adapter.parent_index_name
+        destination_index = adapter.primary.index_name
```

Comment:
We only have `primary` and `secondary` in multiplexed indices. So destination index should be `adapter.index_name`